### PR TITLE
[spec/interpreter/test] Use LEB128 for saturating truncation opcodes

### PR DESCRIPTION
**Pull request by rossberg**
As agreed upon in yesterdays CG meeting.

### CODE REVIEW

File: interpreter/exec/int.ml [hunk 1]
```diff
@@ -221,6 +223,9 @@ struct
   let ge_s x y = x >= y
   let ge_u x y = cmp_u x (>=) y
 
+  let to_int_s = Rep.to_int
+  let to_int_u i = Rep.to_int i land (Rep.to_int Rep.max_int lsl 1) lor 1
```
Comment (binji) [inline interpreter/exec/int.ml:227]: I'm a bit surprised this isn't implemented anywhere else... or is it that the other implementations depend on the int module?

Comment (rossberg) [reply interpreter/exec/int.ml:227]: Yeah, so was I. I think we never needed this for plain int.

**Comment by rossberg** [comment]
Hm, what's the issue with Travis, and why don't I see a "Restart Build" button?

**Comment by ngzhian** [comment]
I had a similar issue with "Restart Build" not showing up, https://stackoverflow.com/a/17624403 seems like I didn't have the right perms or something. In the end I pushed an empty commit, then in the "Rebase and squash" i just removed the description of the empty commit.

**Comment by rossberg** [comment]
@ngzhian, the Internet suggested closing & reopening the PR as the simplest workaround.

**Pull request merged**